Dockerfile

```diff
@@ -43,9 +43,9 @@ RUN wget --no-check-certificate -O master.zip https://github.com/wolfcw/libfaket
     cd libfaketime-master && make install && cd ..
 
 # Grafana
-RUN wget https://dl.grafana.com/oss/release/grafana_6.1.6_amd64.deb ;\
-    dpkg -i grafana_6.1.6_amd64.deb ;\
-    rm grafana_6.1.6_amd64.deb
+RUN wget https://dl.grafana.com/oss/release/grafana_6.4.3_amd64.deb ;\
+    dpkg -i grafana_6.4.3_amd64.deb ;\
+    rm grafana_6.4.3_amd64.deb
```

Comment:
looking at https://grafana.com/grafana/download
there is at least `6.4.5` which we should use here instead.

but if we jumped 3 minor versions, are there any particular reasons we could not jump 3 more to use `6.7.4` ?

if we feel adventurous (and we have to retest all usage scenarios for this image anyway), we could also consider/try using `7.2.2`, wdyt?

Reply:
6.4.5 sounds good.  I have created a RaspberryPi-based Graphite/Grafana instance at home, using 7.2.2, and the interface is so different that it will take a lot of time to get used to it.  I would stay on the 6.x branch for now.  Maybe later we can update to 6.7.x.